pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/kit/KitDaoImpl.java

```diff
@@ -369,14 +468,31 @@ public Integer deleteKitRequest(Long kitRequestId) {
         });
 
         if (simpleResult.resultException != null) {
-            throw new DsmInternalError(String.format("Error deleting kit request with id: %d", kitRequestId),
-                    simpleResult.resultException);
+            throw new RuntimeException("Error deleting kit request with id: " + kitRequestId, simpleResult.resultException);
```

Comment:
I think DSMInternalError is more accurate for here since we are moving on from RuntimeExcpetions